courses/free-courses-hi.md

```diff
@@ -47,6 +47,7 @@
 * [All C Concepts \| Hindi](https://www.youtube.com/playlist?list=PL7ersPsTyYt1d8g5qaxbE6sjWDzs4D_1v) - Saurabh Shukla
 * [C Language Tutorial For Beginners In Hindi (With Notes)](https://www.youtube.com/watch?v=ZSPZob_1TOk) - CodeWithHarry
 * [C Language Tutorials In Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9aiXlHcLx-mDH1Qul38wD3aR) - CodeWithHarry
+* [C Programming Tutorial in English](https://www.youtube.com/watch?v=KJgsSFOSQv0) - Mike Dane (freeCodeCamp)
```

Comment:
is it long enough to not be a playlist?

Reply:
It is included in this playlist: https://www.youtube.com/watch?v=KJgsSFOSQv0&list=PLWKjhJtqVAbmUE5IqyfGYEYjrZBYzaT4m but I think that this resource should not be added in this file because it is in English.

In the correct file it is already added:
https://github.com/EbookFoundation/free-programming-books/blob/d6633c1f42ea59d49761c533b4810693fa0ea8fc/courses/free-courses-en.md#L221